pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -3,159 +3,203 @@
-#pushFlushMaxPoints=40000
-
-## Milliseconds between flushes to the Wavefront servers. Typically 1000.
-#pushFlushInterval=1000
-
-## Limit pps rate at the proxy. Default: do not throttle
-#pushRateLimit=20000
+## DDI/Relay endpoint: in environments where no direct outbound connections to Wavefront servers are possible, you can
+## use another Wavefront proxy that has outbound access to act as a relay and forward all the data received on that
+## endpoint (from direct data Ingestion clients and/or other proxies) to Wavefront servers.
+## This setting is a comma-separated list of ports. (Default: none)
+#pushRelayListenerPorts=2978
 
-## Max number of burst seconds to allow when rate limiting to smooth out uneven traffic.
-## Set to 1 when doing data backfills. Default: 10
-#pushRateLimitMaxBurstSeconds=10
-
-## Max number of points that can stay in memory buffers before spooling to disk. Defaults to 16 * pushFlushMaxPoints,
-## minimum allowed size: pushFlushMaxPoints. Setting this value lower than default reduces memory usage but will force
-## the proxy to spool to disk more frequently if you have points arriving at the proxy in short bursts.
-#pushMemoryBufferLimit=640000
+## Comma-separated list of ports to listen on for OpenTSDB formatted data (Default: none)
+#opentsdbPorts=4242
 
-## If there are blocked points, how many lines to print to the log every 10 flushes. Typically 5.
-#pushBlockedSamples=5
+## Comma-separated list of ports to listen on for HTTP JSON formatted data (Default: none)
+#jsonListenerPorts=3878
 
-# The push log level determines how much information will be printed to the log.
-#   Options: NONE, SUMMARY, DETAILED. Typically SUMMARY.
-pushLogLevel=SUMMARY
+## Comma-separated list of ports to listen on for HTTP collectd write_http data (Default: none)
+#writeHttpJsonListenerPorts=4878
 
-## The validation level keeps certain data from being sent to Wavefront.
-##   We strongly recommend keeping this at NUMERIC_ONLY
-##   Options: NUMERIC_ONLY, NO_VALIDATION.
-#pushValidationLevel=NUMERIC_ONLY
+################################################# DATA PREPROCESSING ###################################################
+## Path to the optional config file with preprocessor rules (advanced regEx replacements and whitelist/blacklists)
+#preprocessorConfigFile=/etc/wavefront/wavefront-proxy/preprocessor_rules.yaml
 
 # When using the Wavefront or TSDB data formats the Proxy will automatically look for a tag named
 # source= or host= (preferring source=) and treat that as the source/host within Wavefront.
-# customSourceTags is a comma separated, ordered list of additional tag keys to use if neither
+# customSourceTags is a comma-separated, ordered list of additional tag keys to use if neither
 # source= or host= is present
 customSourceTags=fqdn, hostname
 
 ## The prefix should either be left undefined, or can be any  prefix you want
 ## prepended to all data points coming through this proxy (such as 'prod').
```

Comment:
Maybe instead of "(such as 'prod)'" say"For example, you could add `production` if your production servers send traffic to this proxy. 
(Or maybe just make the example 'prod' and rule 'prefix=production' match